packages/blade/src/components/Amount/__tests__/Amount.web.test.tsx

```diff
@@ -194,4 +204,19 @@ describe('<Amount />', () => {
       ],
     });
   });
+
+  AMOUNT_SUFFIX_TEST_SET.forEach((item) => {
+    it(`should render ${item.output} in Amount for value:${item.value} & suffix:${item.suffix}`, () => {
+      const { getByTestId } = renderWithTheme(
+        <I18nAmountWrapper
+          value={item.value}
+          suffix={item.suffix}
+          testID="amount-test"
```

Comment:
why are we using test id? we shall use get by text isn't it? that's how we validate how will the text be read by our users

Reply:
`getByText` can't be used here as the text is divided across multiple elements. For eg,
```
<p>$</p>
<p>1000</p>
```
Therefore, getByTestId was used from parent to find if there's any text content that matches the output

Reply:
Either way works. 

getByText(item.output) vs getByTestId("amount.test").toHaveText(item.output)

Reply:
yes but would prefer to refrain from using test ids as with these tests and RTL we are trying to mimic our end users

Reply:
@tarun-khanna differences: https://stackoverflow.com/a/65001155

approving this for now but the above justifies the reason